### PR TITLE
Convert drawables to svg and use resource resolution for light/dark theming

### PR DESCRIPTION
**Pull request by Abestanis**
This converts all of our drawables to svgs. We can do this because svg drawables are supported from Android API `21` onward, which is our current minimum target sdk version.
By converting to svg we reduce the size of the application and make it look sharp regardless of the screen resolution.
We can't remove the `launch_foreground.png` yet, because gradient color is not supported below API `24`.

This also removes the different resources for light and dark themes and the system brightness listener in `AudioHandler`, and instead relies on the automatic themed resource lookup, which get's triggered automatically when the system theme changes. This part is really hard to test, because most systems apply their own theming for media control buttons (e.g. in the Android emulators, they are always white). I observed the switching of colors on my real phone, but if you have a setup that you used to test this when developing it originally, please make sure to test with these changes again.

For context, this is part of my work towards a home screen widget, I want to reuse these drawables.

### CODE REVIEW

File: lib/logic/player/music_player.dart [hunk 1]
```diff
@@ -173,7 +173,7 @@ class _BrowserParentProvider {
 }
 
 @visibleForTesting
-class AudioHandler extends BaseAudioHandler with SeekHandler, WidgetsBindingObserver {
```
[the remark this answers is not in the document]
Comment (Abestanis) [reply lib/logic/player/music_player.dart:176]: Ah, sorry, I didn't see the warning among the others in this file. We might want to upgrade the severity of the `override_on_non_overriding_member` lint rule.

File: android/app/src/main/res/drawable/round_music_note.xml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<vector android:height="48dp"
+    android:viewportHeight="24" android:viewportWidth="24"
+    android:width="48dp" xmlns:android="http://schemas.android.com/apk/res/android">
+    <path android:fillColor="@color/buttonForeground" android:pathData="M12,5v8.55c-0.94,-0.54 -2.1,-0.75 -3.33,-0.32 -1.34,0.48 -2.37,1.67 -2.61,3.07 -0.46,2.74 1.86,5.08 4.59,4.65 1.96,-0.31 3.35,-2.11 3.35,-4.1V7h2c1.1,0 2,-0.9 2,-2s-0.9,-2 -2,-2h-2c-1.1,0 -2,0.9 -2,2z"/>
```
[the remark this answers is not in the document]
Comment (Abestanis) [reply android/app/src/main/res/drawable/round_music_note.xml:4]: Ah, you mean this one, right?
https://github.com/nt4f04uNd/sweyer/blob/d35a51d623328354e74c6568ab782a08bad44972/lib/logic/player/music_player.dart#L659

**Comment by Abestanis** [comment]
Ok, this is pretty close:

| Before   | image |
|----------|---|
| `1.0.8`     | image |
| **After** | image |

**Comment by nt4f04uNd** [comment]
> Yeah, those are the two svgs I had to recreate by hand, I tried to make them exactly the same but evidently didn't succeed.

Could you not just use an existing SVG for that?

**Comment by Abestanis** [comment]
> Could you not just use an existing SVG for that?

I didn't realize those existed as SVGs already. If you can point me at them then I can definitely import them.

**Comment by nt4f04uNd** [comment]
[Loop icon](https://github.com/nt4f04uNd/sweyer/blob/master/static_assets/edit/loop_on.svg)

[Logo foreground](https://github.com/nt4f04uNd/sweyer/blob/master/static_assets/edit/logo/logo_foreground.svg)

**Comment by nt4f04uNd** [comment]
(it looks like we no longer need loop icon, so we can delete it)

**Comment by Abestanis** [comment]
Ah, i looked in the `assets` folder, but not in `static_assets`.

> (it looks like we no longer need loop icon, so we can delete it, I like the new one more)

👍

**Comment by Abestanis** [comment]
Do you want me to put the two `loop_one` SVGs which I created as a source for the drawables into the `static_assets` folder? I'm not sure how easy it is to convert the drawable back into an SVG, if we would ever need to.

**Comment by nt4f04uNd** [comment]
No, we don't need that

**Comment by Abestanis** [comment]
Looks like I cant use the foreground logo unfortunately, it contains some SVG elements that are not supported in Android drawables:
```
ERROR @ line 14: <filter> is not supported
ERROR @ line 15: <feFlood> is not supported
ERROR @ line 16: <feBlend> is not supported
ERROR @ line 18: Semitransparent mask cannot be represented by a vector drawable
```

**Comment by nt4f04uNd** [comment]
Can you just copy the gradient values from it then?

**Review by nt4f04uNd — APPROVED** [review]
Actually, it's not easly doable, it seems. LGTM then

**Comment by Abestanis** [comment]
> it's not easly doable

Yeah, I tried as well, but no success.
Do you just want to delete `static_assets/edit/loop_on.svg`, I forgot to push the commit.

**Pull request merged**